core/commands/object/diff.go

```diff
@@ -1,18 +1,21 @@
 package objectcmd
 
 import (
-	"bytes"
 	"fmt"
 	"io"
 
-	cmds "github.com/ipfs/go-ipfs/commands"
-	e "github.com/ipfs/go-ipfs/core/commands/e"
+	cmdenv "github.com/ipfs/go-ipfs/core/commands/cmdenv"
 	coreiface "github.com/ipfs/go-ipfs/core/coreapi/interface"
 	"github.com/ipfs/go-ipfs/dagutils"
 
+	cmds "gx/ipfs/QmSXUokcP4TJpFfqozT69AVAYRtzXVMUjzQVkYX41R9Svs/go-ipfs-cmds"
 	cmdkit "gx/ipfs/Qmde5VP1qUkyQXKCfmEUA7bP64V2HAptbJ7phuPp7jXWwg/go-ipfs-cmdkit"
 )
 
+const (
+	verboseOptionName = "verbose"
```

Comment:
Is there a reason for this?

Nothing wrong with, but it's not very consistent as we don't use contacts for any of the other options in this file.

Personally, I would err on the side of not using the constant, but others may disagree.

Reply:
@kevina `version` is used in two places, so I use the constant.

Reply:
So is almost every other option, or am I missing something.

In any case it not that big of a deal.

Reply:
Most of the commands have actually been changed to use constants for these.